cmd/run.go

```diff
@@ -42,50 +42,54 @@ var runCmd = &cobra.Command{
 
 func runAndAttach(projectName string, targetOS string) {
 	cmdApp := exec.Command(dotSlash + filepath.Join("desktop", "build", "outputs", targetOS, projectName))
-	cmdApp.Stderr = os.Stderr
+	cmdFlutterAttach := exec.Command("flutter", "attach")
 
 	// Create stdout, streams to parse the debug-uri of the flutter app.
 	// debug-uri is used for hotreloading.
-	stdoutApp, err := cmdApp.StdoutPipe()
-	if err == nil {
+	stdoutApp, errout := cmdApp.StdoutPipe()
+	stderrApp, errerr := cmdApp.StderrPipe()
+
+	if errout == nil && errerr == nil {
 		re := regexp.MustCompile("(?:http:\\/\\/)[^:]*:50300\\/[^\\/]*\\/")
 
+		// Non-blockingly read the stdout to catch the debug-uri
 		go func(reader io.Reader) {
 			scanner := bufio.NewScanner(reader)
-			debugURIFound := false
 			for scanner.Scan() {
-				fmt.Println(scanner.Text()) // defualt stdout
-				if !debugURIFound {
-					match := re.FindStringSubmatch(scanner.Text())
-					if len(match) == 1 {
-						debugURIFound = true
-						go startHotReloadProcess(buildTargetMainDart, match[0])
-					}
+				match := re.FindStringSubmatch(scanner.Text())
+				if len(match) == 1 {
+					startHotReloadProcess(cmdFlutterAttach, buildTargetMainDart, match[0])
+					return
 				}
 			}
 		}(stdoutApp)
 	} else {
-		fmt.Printf("unable to parse flutter debuger'%s' failed with error: %v Hotreload disabled\n", projectName, err)
-		cmdApp.Stdout = os.Stdout
+		fmt.Printf("hover: unable to parse flutter debuger'%s' failed with error: %v %v Hotreload disabled\n", projectName, errout, errerr)
```

Comment:
Not just hotreload is disabled, we completely exit. So "Hotreload disabled" is a bit of an understatement and can be removed, the user will notice that hover exited.